navigation/Navigation.jsx

```diff
@@ -21,12 +24,23 @@ export const Navigation = () => {
   const setAuthToken = async () => {
     const authToken = await SecureStore.getItemAsync('Authorization');
     dispatch(setToken(authToken));
+    setTimeout(() => {
+      dispatch(setGlobalLoader(false));
+    }, 1000);
```

Comment:
Można to kiedyś wydzielić do pliku konfiguracyjnego aplikacji np. `config.js` ze zmiennymi.

Reply:
pisz normalne komentarze